internal/vds/vds_test.go

```diff
@@ -42,11 +42,13 @@ func toFloat32(buf []byte) (*[]float32, error) {
 }
 
 func TestMetadata(t *testing.T) {
-	expected_format := 3
-	expected_axis := []Axis{
-		{ Annotation: "Inline",    Min: 1,  Max: 5,  Samples: 3, Unit: "unitless" },
-		{ Annotation: "Crossline", Min: 10, Max: 11, Samples: 2, Unit: "unitless" },
-		{ Annotation: "Sample",    Min: 4,  Max: 16, Samples: 4, Unit: "ms"       },
+	expected := Metadata{
+		Axis: []{
```

Comment:
Also this doesn't build?

Reply:
I think I got a struct mid-implementation

Reply:
and another STROKE when writing the previous comment. Jeeez...